x-pack/plugins/alerts/common/builtin_action_groups.ts

```diff
@@ -6,13 +6,13 @@
 import { i18n } from '@kbn/i18n';
 import { ActionGroup } from './alert_type';
 
-export const RecoveredActionGroup: ActionGroup = {
+export const RecoveredActionGroup: Readonly<ActionGroup> = {
```

Comment:
naming nit: It seems like we use `recovered` everywhere except here where it is `recovery`. What if we renamed the built-in action group `DefaultRecoveredActionGroup` and then named the field on the alert `recoveredActionGroup`

Reply:
I realize I put my comment on the wrong line where it doesn't actually say `recovery` 🤦‍♀️

Reply:
Haha interesting, I just thought of _recovery_ as the verb for becoming Recovered ...which is why the default AG for is _Recovered_. The term `recoveredActionGroup` _could_ work.... but the past tense feels wrong to me 🤔 

I don't know... Having two first languages makes these weird i my head at times.
Any thoughts from someone else in @elastic/kibana-alerting-services ?